appserver/distributions/pom.xml

```diff
@@ -205,6 +208,34 @@
     </build>
     
     <profiles>
+        
```

Comment:
What do these changes have to do with JACC?

Reply:
I added the factored out module to several of those files, and to make the naming consistent and easy to spot as being JACC the inmemory module was renamed to start with JACC, which needed to be updated in these files. The security modules were grouped a bit more tighter together as well. Comments were added were necessary (see original issue), and the odd ordering or inconsistent comment was addressed when looking at these files as well.